src/pages/ReimbursementAccount/BankAccountManualStep.js

```diff
@@ -100,6 +100,7 @@ class BankAccountManualStep extends React.Component {
                         defaultValue={this.props.getDefaultStateForField('routingNumber', '')}
                         keyboardType={CONST.KEYBOARD_TYPE.NUMBER_PAD}
                         disabled={shouldDisableInputs}
```

Comment:
I'd expect disabled to prevent editing .. why do we need both? Can't we make a global fix directly in TextInput?

Reply:
>I'd expect disabled to prevent editing ..

AFAIK [editable](https://reactnative.dev/docs/textinput#editable) is the actual RN prop that disables inputs. 

>why do we need both?

`editable` alone doesn't seem to prevent focusing on the input, which means focused styles are triggered on web.

>Can't we make a global fix directly in TextInput?

We certainly can. I just kept it consistent with these other instances [here](https://github.com/Expensify/App/blob/ae930fbb366b4937db689804af3cc54942c54e06/src/components/DatePicker/index.android.js#L57-L58) and [here](https://github.com/Expensify/App/blob/ae930fbb366b4937db689804af3cc54942c54e06/src/components/DatePicker/index.ios.js#L94-L95), but we could change those as well. Would that be your preference?

Reply:
I agree with @nkuoch here. What do you think about fixing it here? I just tested it and it seems to work well.

```diff
diff --git a/src/components/TextInput/BaseTextInput.js b/src/components/TextInput/BaseTextInput.js
index 87f02ba528..df118a524f 100644
--- a/src/components/TextInput/BaseTextInput.js
+++ b/src/components/TextInput/BaseTextInput.js
@@ -274,6 +274,7 @@ class BaseTextInput extends Component {
                                         }}
                                         // eslint-disable-next-line
                                         {...inputProps}
+                                        editable={!this.props.disabled}
                                         autoCorrect={this.props.secureTextEntry ? false : this.props.autoCorrect}
                                         placeholder={placeholder}
                                         placeholderTextColor={themeColors.placeholderText}
```

Reply:
Sure, I can fix it in BaseTextInput! We might need a bit more logic than that since there are instances where we want to disable it on mobile, but not necessarily on web e.g. [here](https://github.com/Expensify/App/blob/bd11d61876f43809354d722a49e502e6260bfed8/src/components/DatePicker/index.android.js#L57-L58)

Reply:
I'm not familiar with platform specific code so far so I'm curious, wouldn't that code only run on Android?

Reply:
Correct! We have 3 DatePicker components (web, android and ios) each with its own logic - the code I linked above is from the android component and is run on android only.